### ISSUE TITLE
Multiple focus is provided for the Stackviews

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4570]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.4.3 Focus order

**Test Environment:**
**Test URL** http://cx-guidelines.vmware.com/clarity3FullTesting/
**OS** Windows 10
**User Agents/Assistive Technologies** Firefox 76.0

**Description:**
Multiple focus is provided for the Stack views. When the stack view section is focused First time, the section is not interactive i.e. nothing happens when user presses the enter or space key. When the user presses tab key one more time, the same stack view item is focused again using color highlights/shades. At that time when user presses the enter key, the stack view section is expanded.

**Steps to reproduce:**
1. Open the test URL.
2. Using tab key navigation move focus onto one of the collapsed stack view sections (except for static stack-view) .
3. Press enter key and see it get&#039;s expanded.

**Expected result:**
Focused stack view section should be expanded after pressing enter or space key.

**Actual result:**
Nothing happens when user presses the enter or space key i.e. stack view section is not expanded. When user uses tab key one more time, the same stack view item is refocused with change in the color shade/highlight. At that time when user presses the enter key the stack view section is expanded appropriately. Due to this issue keyboard user might get confused that the stack view is not functional with keyboard. Even if the user figures out that it works when the stack view section is refocused by pressing the tab key again, it still adds an extra navigation step in the reading order and makes navigation order redundant and time consuming with tab key navigation.

**Proposed solution:**
Provide single focus for each of the stack view sections, which provides functional capabilities with keyboard interaction.

**Comment by chlane** [comment]
For visual focus indication guidance see https://confluence.eng.vmware.com/display/A11Y/Visual+Focus+Indication.

**Comment by gnomeontherun** [comment]
@chlane We've previously discussed **not** changing default browser behaviors and overriding the focus indicators. This is also a possible problem because not all browsers use this focus indication style. This recommendation changes that stance, and I think we need to be very careful recommending this and understanding the impact of the system.

**Comment by clane** [comment]
@gnomeontherun Yes I've had to change my stance on that and recommend that Clarity provides a custom visual focus indication for the the focused states of its components.  I didn't know that WCAG 2.2 would add a success criterion that would mean that default browser focus outlines do not automatically qualify. 

See  [focus-visible-enhanced](https://www.w3.org/TR/WCAG22/#focus-visible-enhanced) and [this comment from the WCAG](https://github.com/w3c/wcag/issues/1003#issuecomment-580689922).

**Comment by clane** [comment]
****
